large-records-benchmarks/large-records-benchmarks.cabal

```diff
@@ -25,6 +25,10 @@ common defaults
   ghc-options:
       -Wall
 
+      -- The primary goal of this work is to improve the development cycle, so
+      -- we compile with -O0 for the benchmarks.
+      -O0
```

Comment:
But these are benchmarks!

The `-O0` results are not what people actually use. (Neither `-O2`, tbh).

Reply:
We are benchmarking compilation time, not runtime performance.

Reply:
`-O1` affects compilation time as well. GHC works harder after plugin is run. (And plugin affects that as well).

Reply:
Yes, `-O1` does affect compilation time. Hence, if compilation time is a factor, you will compile with `-O0`, and use `-O1` (or -`O2`) only for occasional production builds.

Reply:
I repeat. Use `optimization: False` in `cabal.project.local` file for development. Keep the default as it was meant to be for production. Flags are not necessary for these toggles.

Reply:
This is just _wrong_. This is not a flag you can toggle on or off. The benchmarks _are measuring compilation time with `-O0`_. That is their definition. Toggling this on or off is meaningless.